Cpix/CpixDocument.cs

```diff
@@ -209,12 +216,18 @@ public void Save(Stream stream)
 			}
 		}
 
+		private static void ValidateConstraintsBetweenDrmSystemsAndContentKeys(IEnumerable<DrmSystem> drmSystems, IEnumerable<ContentKey> contentKeys)
+		{
+			if (drmSystems.Select(s => s.KeyId).Except(contentKeys.Select(k => k.Id)).Any())
```

Comment:
I would find this easier to read if you explicitly listed the collections as separate variables.

var referencedKeys = ...
var knownKeys = ...

if (referencedKeys.Except(knownKeys).Any())

Reply:
OK.